### DISCUSSION TITLE
Which Model Analogue?

### DISCUSSION THREAD

**Discussion by friedpa** [jomjol/AI-on-the-edge-device#2553] [General] [answered]
@haverland Hi Frank, could you please let me know, which Model Class you would recommend for the following Watermeter. Thx!

![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/42890695/c418bbb4-ba36-4acf-ad19-6f289f6c3f70)

**Comment by ghost** [comment, marked as answer]
For my watermeter, I use for digits dig-class100-0165. The newer ones are only a bit better ( https://github.com/haverland/Tenth-of-step-of-a-meter-digit/tree/master/output )

For pointers I use ana-class100-0170. All without any error for weeks.
The newer ana-cont-12xx are nearby the ana-class100. Older ana-cont's are not as accurate as the ana-class100, but not really bad too.

<img width="448" alt="image" src="https://github.com/jomjol/AI-on-the-edge-device/assets/412645/f6dcd3e2-edc9-47fc-b587-f527b4188c07">

**Comment by friedpa** [comment]
Thanks!
